modin/pandas/test/test_series.py

```diff
@@ -4393,13 +4464,7 @@ def test_cat_codes(data):
     df_equals(modin_result, pandas_result)
 
 
-@pytest.mark.parametrize(
-    "set_min_partition_size",
-    [1, 2],
-    ids=["four_partitions", "two_partitions"],
-    indirect=True,
-)
-def test_cat_codes_issue5650(set_min_partition_size):
```

Comment:
[`set_min_partition_size`](https://github.com/modin-project/modin/blob/aae34e724f0c471b5079339eb28d187fed9d35e5/modin/conftest.py#L552) is a pytest fixture changing modin's configuration for a specific test, can you please revert it back?